AssetInformationApi/V1/Controllers/AssetInformationApiController.cs

```diff
@@ -201,5 +202,15 @@ public async Task<IActionResult> PatchAssetAddress([FromRoute] EditAssetByIdRequ
 
             return null;
         }
+
+        private void RestrictAccessToDisallowedEmail(GetAssetByIdRequest req)
+        {
+            // This is an exceptional case where a specific user is not allowed to access a specific asset
+            // See HPT-641 for more information
```

Comment:
Thinking this over, this board will likely get archived into the future. Would be better to link some Google Doc instead.